data-prepper-plugins/blocking-buffer/README.md

```diff
@@ -0,0 +1,20 @@
+# Blocking Buffer
+
+This is a buffer based off `LinkedBlockingQueue` bounded to the specified capacity. One can read and write records with specified timeout value.
+
+## Configuration
+- buffer_size => An `int` representing max number of records the buffer accepts.
+- batch_size => An `int` representing max number of records the buffer returns on read.
+
+## Usages
+Example `.yaml` configuration
+```
+buffer:
+    - bounded_blocking:
+```
+*Note*: *By default, Data Prepper uses only one buffer. the `bounded_blocking` buffer, so this section in the `.yaml` need not be defined unless one wants to mention a custom buffer or tune the buffer settings* 
+
```

Comment:
Although there is no custom metrics, we still need to add the Metrics section to point out the buffer inherits common metrics in `AbstracBuffer`.